### PR TITLE
fix(preview): Don't log a warning on missing exif orientation metadata

### PR DESCRIPTION
**Pull request by anthonybilinski**
libexif returns 0 for the orientation when orientation metadata isn't present.
Treat this the same as 1, i.e. no orientation change.

- [x] Commits follow our [git commit guidelines](https://github.com/qTox/qTox/blob/master/CONTRIBUTING.md#git-commit-guidelines)

---
This change is [Reviewable](https://reviewable.io/reviews/qtox/qtox/6160)

### CODE REVIEW

**Comment by anthonybilinski** [comment]
I've created https://github.com/libexif/libexif/issues/46 to better understand if we're misusing the libexif API, or if this is an upstream issue that we need to work around.

**Comment by sudden6** [comment]
Should we wait for an answer of the libexif people until merging this, or merge it now?

**Comment by anthonybilinski** [comment]
This is a regression from our old behaviour, see here https://github.com/qTox/qTox/pull/6143/files#diff-a775ed8850c5781a2c749c3cafbe4e10L530, and is also how orientation seems to be handled from libexit wherever I look online. IMO including this workaround until (if ever) something is changed upstream seems reasonable.

I'd be fine merging this without waiting on the upstream issue.

**Review by sudden6 — APPROVED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/qtox/qtox/6160)* status: 0 of 1 LGTMs obtained

**Review by sudden6 — APPROVED** [review]
:lgtm_strong:

> *[Reviewable](https://reviewable.io/reviews/qtox/qtox/6160#-:-M8RnprZ3uxfE8HlEZHJ:b-jbsf8z)* status: :shipit: complete! 1 of 1 LGTMs obtained

**Pull request merged**
